client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/rates-step/test/list.js

```diff
@@ -1,363 +0,0 @@
-/** @format */
```

Comment:
The `<ShippingRates/>` component has been vastly simplified in this PR. It no longer renders anything conditionally. Do you folks think it's still worth it to keep some of these tests? For what I've seen, they mostly tested the logic for higind the dropdowns on empty rates, displaying multiple errors, etc (which is no longer relevant).

Reply:
I'll wait to make a final call to see how this may affect things: https://github.com/Automattic/wp-calypso/pull/26979/files/1518bba22db984af895c400d6112046e9abe45e3#r215014901

In general though, I'm not a big fan of deleting tests unless all the code it tested is also being deleted. How hard is it to adapt the tests to the new behavior?